components/omega/src/ocn/Tendencies.cpp

```diff
@@ -38,6 +39,49 @@ int Tendencies::init() {
    Tendencies::DefaultTendencies =
        create("Default", DefHorzMesh, NVertLevels, NTracers, &TendConfig);
 
+   // Check if use the customized tendencies
+   bool UseCustomTendency = false;
+   Err = TendConfig.get("UseCustomTendency", UseCustomTendency);
+   if (Err != 0) {
+      LOG_ERROR("Tendencies:: UseCustomTendency not found in Config");
+      return Err;
+   }
```

Comment:
It isn't clear to me why this isn't a critical error:
```suggestion
   if (Err != 0) {
      LOG_CRITICAL("Tendencies: UseCustomTendency not found in Config");
   }
```
If we aren't going to initialize tendencies, I think we should stop the code.